# 油门系统控制状态
int8 throttle_sys_status

# 制动系统控制状态
int8 ebs_status

# 油门踏板开度（%）
int8 acc_pedal_value

# 制动踏板开度（%)
int8 brake_pedal_value

float32 cruise_speed

# 发动机转速(rpm)
float32 engine_speed

# 发动机转矩(N.m)
float32 engine_torque

int8 gear_number

float32 ego_speed

float32 ego_accel

# 总质量 (kg)
float32 ego_mass

float32 engine_torque_limit

# 缓速器状态，自动再生状态，瞬间油耗，小计油耗，小计里程，总计油耗，总计里程，胎压数值

# 风速，雨量，温度，地面摩擦系数